codecov.yml

```diff
@@ -0,0 +1,38 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 5%
+        base: auto
+        flags:
+          - unit
+        # advanced
+        branches:
+          - master
+        if_not_found: success
+        if_ci_failed: error
+        informational: false
+        only_pulls: false
+
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: false  # if true: only post the comment if coverage changes
+  require_base: no        # [yes :: must have a base report to post]
+  require_head: no        # [yes :: must have a head report to post]
+  branches:               # branch names that can post comment
+    - "master"
```

Comment:
All branches except `master` should be available here In order to enable comments in PR according to [source](https://docs.codecov.io/docs/pull-request-comments#branches). The first report should appear on this PR.

Reply:
I was curious why there are no comments in this PR. Let's try this 👍